Start_applications.bat

```diff
@@ -1,10 +1,11 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-call %~dp0get_conda_exec.bat
+call "%~dp0get_conda_exec.bat"
 if !errorlevel! neq 0 (
   exit /B !errorlevel!
 )
 
-call !MY_CONDA_EXE! activate geoapps && jupyter notebook geoapps\index.ipynb
+set MY_CONDA=!MY_CONDA_EXE:"=!
```

Comment:
Is this line missing a matching quote " character?

Reply:
Loooks like it, nice catch

Reply:
no, that is the syntax to remove existing quote in the path... before I wrap again the variable in quote when calling conda
(Do not ask my why, but conda fails if I just leave the quotes as is in the variable)

PS: did I mentioned already that I hate batch script?

Reply:
haha yes I actually wasn't sure. It definitely looks weird.